ykrt/src/compile/jitc_yk/codegen/x86_64/mod.rs

```diff
@@ -413,7 +419,7 @@ impl<'a> X64CodeGen<'a> {
             let op = &args[i];
             // We can type check the static args (but not varargs).
             debug_assert!(
-                i >= fty.num_args() || op.type_(self.m) == fty.arg_type(self.m, i),
+                i >= fty.num_args() || self.m.type_(op.ty_idx(self.m)) == fty.arg_type(self.m, i),
```

Comment:
Should a follow up commit rename `type_` to `ty`, as per our new conventions?

Reply:
Good point. Yes!

Reply:
Well, follow-up PR.

Reply:
OK